frontend/src/metabase/components/Radio.styled.jsx

```diff
@@ -7,9 +7,15 @@ import { color, lighten } from "metabase/lib/colors";
 const COLOR_SCHEMES = {
   admin: {
     main: () => color("accent7"),
+    button: () => color("brand"),
+  },
+  accent7: {
```

Comment:
Not sure why I added `button: () => color("brand"),` for the `admin` variant. Probably because I did not need it for the permission pages and kept the default value. But it seems like we can merge admin and accent7 variants, what do you think?